src/systemctl/systemctl-edit.c

```diff
@@ -358,7 +358,7 @@ static int run_editor(char **paths) {
                 if (n_editor_args > 0)
                         execvp(args[0], (char* const*) args);
 
-                FOREACH_STRING(p, "editor", "nano", "vim", "vi") {
+                FOREACH_STRING(p, "editor", "nano", "emacs", "vim", "vi") {
```

Comment:
I don't think we should start an editor that pops up a graphical window. This is confusing to users. The editor must be a "text editor" that shows up in the current terminal and blocks it until exit.
`emacs -nw` would be acceptable and better.

But then this loop would have to be changed…